Zhuang/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pip install -r requirements.txt
```

Comment:
```suggestion
```

There's no `requirements.txt` on this branch so this will fail. You've removed `requirements.txt` purposefully, so you need to update the instructions as well.

Reply:
fix it